src/browser.py

```diff
@@ -205,8 +206,9 @@ def getCCodeLang(lang: str, geo: str) -> tuple:
             try:
                 nfo = ipapi.location()
             except RateLimited:
-                logging.warning("Returning default", exc_info=True)
-                return "en", "US"
+                geo = Utils.loadConfig("config-private.yaml").get("default_geolocation", "US").upper()
```

Comment:
If this is explicitly set, I think we should prefer it to the location based on IP.